backend/manager/modules/utils/src/main/java/org/ovirt/engine/core/utils/ovf/OvfOvirtReader.java

```diff
@@ -260,10 +260,10 @@ public void buildDisk() {
     protected void readDisk(XmlNode node, DiskImage image) {
         super.readDisk(node, image);
 
-        if (!StringUtils.isEmpty(node.attributes.get("ovf:size").getValue())) {
+        if (node.attributes.get("ovf:size").getValue() != null && !StringUtils.isEmpty(node.attributes.get("ovf:size").getValue())) {
```

Comment:
Isn't `StringUtils.isEmpty()` handling `null` values correctly?

https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/index.html

Reply:
Yeah, the return of the get from the attributes needs to be checked
@liranr23 how about adding a unit test with the xml (ovf) that was shared on the users list?

Reply:
> Isn't `StringUtils.isEmpty()` handling `null` values correctly?
> 
> https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/index.html

Yes, now it makes more sense as the attribute may result in null.